lib/content.js

```diff
@@ -1,80 +1,90 @@
-const { simplePost, populateSimplePost } = require('./content-structure');
+const request = require("request-promise-native")
 
 module.exports = {
   createEntry,
   getLatestVersion,
   updateEntry,
   publishEntry
-};
+}
 
 // Promise<[createdEntryId: int, createdEntryVersion: int]>
-async function createEntry ({ cmaToken, spaceId }) {
+async function createEntry({
+  cmaToken,
+  spaceId,
+  contentTypeId,
+  populateContent
+}) {
   const options = {
-    method: 'POST',
+    method: "POST",
     url: `https://api.contentful.com/spaces/${spaceId}/environments/master/entries`,
     token: cmaToken,
     headers: {
       Authorization: `Bearer ${cmaToken}`,
-      'Content-Type': 'application/json',
-      'X-Contentful-Content-Type': simplePost.contentTypeId
+      "Content-Type": "application/json",
+      "X-Contentful-Content-Type": contentTypeId
     },
-    body: JSON.stringify(populateSimplePost())
-  };
+    body: JSON.stringify(populateContent())
+  }
 
-  const raw = await request(options);
-  const parsed = JSON.parse(raw);
+  const raw = await request(options)
+  const parsed = JSON.parse(raw)
 
-  return [parsed.sys.id, parsed.sys.version];
+  return [parsed.sys.id, parsed.sys.version]
 }
 
 // Promise<updatedEntryVersion: int>
-async function updateEntry ({ cmaToken, spaceId, entryId, version }) {
+async function updateEntry({
+  cmaToken,
+  spaceId,
+  entryId,
+  version,
+  populateContent
+}) {
   const options = {
-    method: 'PUT',
+    method: "PUT",
     url: `https://api.contentful.com/spaces/${spaceId}/environments/master/entries/${entryId}`,
     token: cmaToken,
     headers: {
       Authorization: `Bearer ${cmaToken}`,
-      'Content-Type': 'application/json',
-      'X-Contentful-Version': version
+      "Content-Type": "application/json",
+      "X-Contentful-Version": version
     },
-    body: JSON.stringify(populateSimplePost())
-  };
+    body: JSON.stringify(populateContent())
+  }
 
-  const raw = await request(options);
-  const parsed = JSON.parse(raw);
+  const raw = await request(options)
+  const parsed = JSON.parse(raw)
 
-  return parsed.sys.version;
+  return parsed.sys.version
 }
 
 // Promise<>
-async function publishEntry ({ cmaToken, spaceId, entryId, version }) {
+async function publishEntry({ cmaToken, spaceId, entryId, version }) {
   await request({
-    method: 'PUT',
+    method: "PUT",
     url: `https://api.contentful.com/spaces/${spaceId}/environments/master/entries/${entryId}/published`,
     token: cmaToken,
     headers: {
       Authorization: `Bearer ${cmaToken}`,
-      'Content-Type': 'application/json',
-      'X-Contentful-Version': version
+      "Content-Type": "application/json",
+      "X-Contentful-Version": version
     }
-  });
+  })
 }
 
 // Promise<version: int>
-async function getLatestVersion ({ cmaToken, spaceId, entryId }) {
+async function getLatestVersion({ cmaToken, spaceId, entryId }) {
   const options = {
-    method: 'GET',
+    method: "GET",
```

Comment:
Can you disable prettify for this please? :)

Reply:
I can unprettify at once before merging